assets/src/apps/delivery/components/preview-tools/Inspector.tsx

```diff
@@ -69,13 +69,22 @@ const Inspector: React.FC<InspectorProps> = ({ currentActivity }) => {
             (activity) => !!activity.content.partsLayout.find((p: any) => p.id === part),
           );
           if (ownerActivity.id === activity.id) {
-            const next = { ...col, ...statePuff[`${activity.id}|stage`] };
```

Comment:
I saw an issue with the inspector removing MCQ  variables from list after performing trigger check. So fixed this and also tested Plate tectonics- inspector getting out of sync to make sure both works.